package.json

```diff
@@ -110,7 +110,7 @@
     "react": "^16.8.0 || ^17.0.0 || ^18.0.0"
   },
   "engines": {
-    "node": ">=16.9.1",
```

Comment:
Hey @tizmagik, typically dropping support for older node versions is considered a breaking change, but this got pulled in on a minor version.
It also looks like this is dropping support for node 16 as well, which isn't in the release notes (it only mentions node 12)

as a result, we're seeing this on Node 16.19.1 now

```
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: 'react-tracking@9.3.0',
npm WARN EBADENGINE   required: { node: '>=18', npm: '>=8.0.0' },
npm WARN EBADENGINE   current: { node: 'v16.19.0', npm: '8.19.3' }
npm WARN EBADENGINE }
```